python_modules/dagster/dagster_tests/asset_defs_tests/test_asset_group.py

```diff
@@ -43,6 +43,9 @@ def _all_asset_keys(result):
 @pytest.fixture(autouse=True)
 def check_experimental_warnings():
     with warnings.catch_warnings(record=True) as record:
+        # turn off any outer warnings filters
+        warnings.resetwarnings()
```

Comment:
I might not be understanding, but does turning off external filters actually end up testing for a different warning profile than a user gets in a default environment?

Reply:
Yes, because external filters are set by our test infrastructure, which is not present in user environments.

https://github.com/dagster-io/dagster/blob/master/pyproject.toml#L165